<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPatient</class>
 <widget class="QDialog" name="addPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 117, 143);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatname">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatname">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatdetails">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Patient Details</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatphone">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatphone">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatgender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckMale">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckFemale">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckOther">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckImmersion">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Immersion</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckContact">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatopmode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>306</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Operating Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckPhakic">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>255</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Phakic</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatopmethod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Operating Method</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckApakic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>255</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Aphakic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckDense">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>255</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Dense</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckRight">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatopeye">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>366</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Operating Eye</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckLeft">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 25px;
    color: rgb(255, 255, 255);
    font-weight: bold;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: black;
    border: 1px solid gray;
}

QCheckBox::indicator:checked {
    background-color: black;
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border: 1px solid gray;
}
</string>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkleft">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KeratoMeter - LEFT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkl1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KL1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkl1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkl2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KL2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkl2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkr2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KR2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkright">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KeratoMeter - RIGHT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkr1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KR1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatkr">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>KR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkr">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkr2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepatkr1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButPatSave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButPatBack">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
